robot_hw:
  loop_frequency: 500
  cycle_time_error_threshold: 0.002
  power_limit: 6
  contact_threshold: 40

